.github/workflows/deploy-pages.yaml

```diff
@@ -55,3 +56,37 @@ jobs:
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         publish_dir: ./docs/_build/html
+        destination_dir: docs  # Publishes to the docs folder
+
+
+jobs:
```

Comment:
Does this work? I would have guessed you can't have two "job" blocks

Reply:
Heads up https://github.com/NREL/floris/actions/runs/13189868647/workflow

Reply:
Trying a tighter integration now